projects/spectator/src/lib/internals/key-parser.ts

```diff
@@ -0,0 +1,52 @@
+export interface ModifierKeys {
+  alt?: boolean;
+  control?: boolean;
+  shift?: boolean;
+  meta?: boolean;
+}
+
+export interface KeyOptions {
+  key: string | false;
+  keyCode: number | false;
+  modifiers: ModifierKeys;
+}
+
+export const parseKeyOptions = (keyOrKeyCode: string | number): KeyOptions => {
+  if (typeof keyOrKeyCode === 'number' && keyOrKeyCode) {
+    return { key: false, keyCode: keyOrKeyCode, modifiers: {} };
+  }
+
+  if (typeof keyOrKeyCode === 'string' && keyOrKeyCode) {
```

Comment:
We have an `isString` [helper](https://github.com/ngneat/spectator/blob/64fc81609ddb81aec37d4b247b1c01e2b2fd2584/projects/spectator/src/lib/types.ts#L23), you can use it. Please alsp add `isNumber` and use it above.